src/fuzzing-service/Taskfile.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+tasks:
+  core-dump-notification-config:
+    summary: Configure core dump notification if running in GCE
+    desc: |
+      Hmm, your system is configured to send core dump notifications to an
```

Comment:
```suggestion
      This section is for when afl2.52b runs in GCE, it may display a message like
      
      Hmm, your system is configured to send core dump notifications to an
```